packages/nuxt/src/app/composables/id.ts

```diff
@@ -0,0 +1,11 @@
+const getUniqueIDKeyPrefix = '$id'
+
+export function getUniqueID(): string
+export function getUniqueID (...args: any): string {
+  const key = args[0]
+  if (typeof key !== 'string') {
+    throw new TypeError(`[nuxt] [getUniqueID] key must be a string: ${key}}`)
+  }
+
+  return getUniqueIDKeyPrefix + key
```

Comment:
Is this functional now?

Reply:
What do you mean - why do you ask?

(personally I think we should probably omit the prefix)

Reply:
Yeah without a prefix should be unique enough...  Added more to the context in https://github.com/nuxt/nuxt/pull/23368#issuecomment-1738799146